src/utils/update/psc.to.beneficial.owner.type.mapper.ts

```diff
@@ -0,0 +1,85 @@
+import { CompanyPersonWithSignificantControl } from "@companieshouse/api-sdk-node/dist/services/company-psc/types";
+import { AddressResource } from "@companieshouse/api-sdk-node/dist/services/confirmation-statement";
+import { BeneficialOwnerGov } from "../../model/beneficial.owner.gov.model";
+import { BeneficialOwnerIndividual } from "../../model/beneficial.owner.individual.model";
+import { BeneficialOwnerOther } from "../../model/beneficial.owner.other.model";
+import { Address, InputDate, yesNoResponse } from "../../model/data.types.model";
+
+export const mapPscToBeneficialOwnerTypeIndividual = (psc: CompanyPersonWithSignificantControl): BeneficialOwnerIndividual => {
+  return {
+    id: psc.links?.self,
+    first_name: psc.nameElements?.forename,
+    last_name: psc.nameElements?.surname,
+    nationality: psc.nationality,
+    second_nationality: undefined,
+    date_of_birth: mapDateOfBirth(psc),
+    is_service_address_same_as_usual_residential_address: isSameAddress(psc?.address) ? yesNoResponse.Yes : yesNoResponse.No,
+    usual_residential_address: mapAddress(psc),
+    service_address: mapAddress(psc),
+    beneficial_owner_nature_of_control_types: undefined,
+    trustees_nature_of_control_types: undefined,
+    non_legal_firm_members_nature_of_control_types: undefined,
+    start_date: psc.notifiedOn as any, // any type not correct but to confirm
```

Comment:
No need to cast to any. But still not known if we can use this here.

Reply:
done